dt/uni-beta/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployed Topology - Beta
+
+If you are looking for information on how to deploy the Beta based DT, then
+please the [README](../../examples/dt/uni-Beta/README.md) in the examples
+directory.
+
+This directory `dt/uni01beta/`, exists so that the
```

Comment:
the directory is  called dt/uni-beta, even thought it could be called uni02beta to keep the alphabetical ordering

Reply:
Thanks, fixed.